operator/gitops/compute/pln-svc-manager/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
This directory will need to be renamed as well.

Reply:
sure, I can change all those instances to `pipeline-service-manager`.